public/scripts/setup.js

```diff
@@ -3,7 +3,6 @@ function initializeUser(access_token, refresh_token){
   var userLastId = $("#lastId").val();
   localStorage.setItem("userLastId", userLastId);
   getTrackForUser(getFourWeeks(), getYesterday());
-  getPlaylists(access_token);
```

Comment:
this snuck back in - `getPlaylists()` takes two arguments, `access_token` and `allCallSongs`, and we don't have `allCallSongs` yet at this point, so it's not yet safe to call `getPlaylists()`. it ends up being called in `getFullHistory()`'s success handler once we've fully populated `allCallSongs`; calling it here just lets you interact with the playlist dropdown before the program has all of the necessary data, and so it allows you to pick a playlist too soon and crash the program.